lib/scenic/adapters/postgres/refresh_dependencies.rb

```diff
@@ -0,0 +1,67 @@
+module Scenic
+  module Adapters
+    class Postgres
+      class RefreshDependencies
+        def self.call(name, adapter, connection)
+          new(name, adapter, connection).call
+        end
+
+        def initialize(name, adapter, connection)
+          @name = name
+          @adapter = adapter
+          @connection = connection
+        end
+
+        def call
+          dependencies.each do |dependency|
+            adapter.refresh_materialized_view(dependency)
+          end
+        end
+
+        private
+
+        attr_reader :name, :adapter, :connection
+
+        DEPENDENCY_SQL = <<~SQL.freeze
```

Comment:
Looks like the `<<~` construct is tripping up CodeClimate. It's using a 2.1 ruby parser for rubocop and gives the error "unexpected token tLSHFT". The other two issues it notes are because the SQL heredoc is being interpreted as code instead of a string. Can you look into that?